### PR TITLE
Added issue template for Az 4.x

### PR DESCRIPTION
**Pull request by dcaro**
## Description

Added a template for issues related to the Az 4.x preview module

## Checklist

- [X] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: .github/ISSUE_TEMPLATE/az4-preview-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+name:  Az 4.x preview bug report 
+about: Report errors or unexpected behaviors specifically for the Az 4.0  module in preview
+title: ''
+labels: 'Az 4.x Preview'
```
Comment (markcowl) [inline .github/ISSUE_TEMPLATE/az4-preview-bug-report.md]: We need to add this label first, right?

Comment (markcowl) [reply .github/ISSUE_TEMPLATE/az4-preview-bug-report.md]: Should also have the 'Azure PS Team' label

Comment (dcaro) [reply .github/ISSUE_TEMPLATE/az4-preview-bug-report.md]: Correct, I have updated the label and created the label.

File: documentation/preview/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Azure PowerShell module 4 preview - User documentation
+
+This article explains how to get started with the preview of the Az 4.x PowerShell module.
+
+## Benefits
+
+With this new module we are introducting several changes to Azure PowerShell. The links below will provide more details for each feature and allow you to submit comments:
+
+- [Add support for Azure profiles](../RFC/RFC0001-Azure-Profiles.md)
```
Comment (markcowl) [inline documentation/preview/README.md]: I think we want to point at the `Feature` documents I created here, and add a section on Feedback that points to the RFCs.

File: documentation/preview/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- KeyVault
+- [Monitor]
+- [Network]
+- Resources
+- ServiceBus
+- Storage
+
+## Feedback
+
+If there is a feature you would like to see in Azure PowerShell, please use the [`Send-Feedback`][SendFeedback] cmdlet, or file an issue in our [GitHub issues][GitHubIssues] page to provide the Azure PowerShell team direct feedback.
```
Comment (markcowl) [inline documentation/preview/README.md]: This is where we should have all of the RFC references (note the RFCs in my PR as well)

File: documentation/preview/README.md (hunk 2 above)
Comment (dcaro) [reply documentation/preview/README.md]: That makes sense, I had this part written before we the `Feature` document exist.

**Comment by dcaro** [comment]
@markcowl PTAL. We'll need to update this with the release notes of each module.

**Pull request merged**